vividus-soft-assert/src/main/java/org/vividus/softassert/issue/KnownIssueChecker.java

```diff
@@ -31,6 +31,7 @@
 
     private ITestInfoProvider testInfoProvider;
     private IIssueStateProvider issueStateProvider;
+    private boolean allowPotentiallyKnownIssues;
```

Comment:
hmm imho enable/disable looks semantically better here

Reply:
We'll wait @valfirst  on this comment :)

Reply:
`detectPotentiallyKnownIssues`?

Reply:
also would you mind updating/adding docs?

Reply:
@valfirst it could be done as separate commit to unblock our guys, or we can wait until the docs are ready?

Reply:
I think they will wait for release, as for docs: you need to add only description or the new property, nothing else